projects/rero/ng-core/src/lib/record/detail/detail.component.ts

```diff
@@ -34,7 +37,25 @@ export class DetailComponent implements OnInit {
   viewComponent: any = null;
 
   /**
-   * Record to display
+   * Object for checking record deletion permission.
+   */
+  canDeleteResult: DeleteRecordStatus = {
+    can: true,
+    message: ''
+  };
+
+  /**
+   * Record can be updated ?
+   */
+  canUpdateResult = true;
```

Comment:
can we uniformize by returning a `ActionStatus`?

Reply:
done